install/0000_90_cluster-version-operator_01_prometheusrolebinding.yaml

```diff
@@ -1,9 +1,9 @@
-apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I think we want this

Reply:
Hah, yes.  Fixed with a1bde66f -> f58dd1c5